### DISCUSSION TITLE
Is it a problem with the default role permissions in the background, or a bug in the program?

### DISCUSSION THREAD

**Discussion by zhuyifang** [directus/directus#7941] [Archive]
I created a data model m1 which mainly consists of two fields
`user_created`
`target_user_id`
They are respectively linked to directus_users.id through many-to-one

When I get` /items/m1? fields =user_created.*,target_user_id.*`

If the currently `logged-in user`=`user_created`, the returned data is all normal

If the `logged-in user` != `user_created`, the returned user_created object is null

Is it a problem with the default role permissions in the background, or a bug in the program?

**Comment by rijkvanzanten** [comment]
Heya! Assuming you have app-access enabled for your user, the default permissions for the user are set to only be able to read their own user profile. It sounds like you might have to open up those permissions a little bit in settings in order to have this request work the way you want it to 👍🏻
